### PR TITLE
Drop SonataCoreBundle

### PR DESCRIPTION
**Pull request by wbloszyk**
## Subject

I am targeting this branch, because it its BC-break.

## Changelog

```markdown
### Removed
- Removed `SonataCoreBundle` 
```

### CODE REVIEW

File: src/Test/XliffValidatorTestCase.php [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+    protected function setUp(): void
+    {
+        $this->loader = new XliffFileLoader();
+    }
+
+    /**
+     * @dataProvider getXliffPaths
+     */
+    public function testXliff($path)
```
Comment (phansys) [inline src/Test/XliffValidatorTestCase.php]:
```suggestion
    public function testXliff(string $path): void
```

File: src/Test/XliffValidatorTestCase.php [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+    /**
+     * @return array List all path to validate xliff
+     */
+    abstract public function getXliffPaths();
+
+    /**
+     * @param string $file The path to the xliff file
+     */
+    protected function validateXliff($file)
```
Comment (phansys) [inline src/Test/XliffValidatorTestCase.php]:
```suggestion
    protected function validateXliff(string $file): void
```

File: src/Test/XliffValidatorTestCase.php [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+            $this->assertTrue(true, sprintf('Successful loading file: %s', $file));
+        } catch (InvalidResourceException $e) {
+            $this->errors[] = sprintf('%s => %s', $file, $e->getMessage());
+        }
+    }
+
+    /**
+     * @param string $path The path to lookup for Xliff file
+     */
+    protected function validatePath($path)
```
Comment (phansys) [inline src/Test/XliffValidatorTestCase.php]:
```suggestion
    protected function validatePath(string $path): void
```

File: src/Test/XliffValidatorTestCase.php [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+            0,
+            $this->errors,
+            sprintf('Unable to parse xliff files: %s', implode(', ', $this->errors))
+        );
+    }
+
+    /**
+     * @return array List all path to validate xliff
+     */
+    abstract public function getXliffPaths();
```
Comment (phansys) [inline src/Test/XliffValidatorTestCase.php]:
```suggestion
    abstract public function getXliffPaths(): array;
```
Then `XliffTest` should be updated too.

File: src/SonataCommentBundle.php [hunk 5]
```diff
@@ -13,7 +13,6 @@
 
 namespace Sonata\CommentBundle;
 
-use Sonata\CoreBundle\Form\FormHelper;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
```
Comment (phansys) [inline src/SonataCommentBundle.php]: CS check [is failing](https://github.com/sonata-project/SonataCommentBundle/pull/179/checks?check_run_id=539182671#step:4:14).

**Comment by greg0ire** [comment]
The build fails, please look into it.

**Comment by wbloszyk** [comment]
Closing this in favor of #185

**Pull request closed without merging**